src/main/java/frc/robot/subsystems/swerve/factories/modules/constants/ModuleConstantsFactory.java

```diff
@@ -1,14 +1,21 @@
 package frc.robot.subsystems.swerve.factories.modules.constants;
 
 import frc.robot.Robot;
+import frc.robot.subsystems.swerve.SwerveName;
 import frc.robot.subsystems.swerve.modules.ModuleConstants;
 
 public class ModuleConstantsFactory {
 
-    public static ModuleConstants create(){
-        return switch (Robot.ROBOT_TYPE){
-            case REAL, REPLAY -> RealModuleConstants.getModuleConstants();
-            case SIMULATION -> SimulationModuleConstants.getModuleConstants();
+    public static ModuleConstants create(SwerveName swerveName) {
+        return switch (swerveName) {
+            case SWERVE -> getSwerveModuleConstants(swerveName);
+        };
+    }
```

Comment:
no need for this now, if we would have 2 swerves we will add (maybe we would create different factories at all for each swerve)

Reply:
The whole point is to write more abstract code so we won't need to change it later. If you delete swerve name usage it defeats its purpose. I don't get the reason for deleting